tests/system_tests_bad_configuration.py

```diff
@@ -112,11 +113,11 @@ def wait_for_unresolvable_host(self):
         with open('../setUpClass/test-router.log', 'r') as router_log:
             log_lines = router_log.read().split("\n")
             expected_errors = [
-                "proton:io Name or service not known",  # Linux
-                "proton:io unknown node or service",  # macOS
+                "Connection to %s" % self.unresolvable_host_name
             ]
             errors_caught = [line for line in log_lines
-                             if any([expected_error in line for expected_error in expected_errors])]
+                             if any([expected_error in line for expected_error in expected_errors])
+                             and "failed" in line]
```

Comment:
If you turn `expected_errors` list above into single `expected_error` value, you can then simplify this into

```suggestion
                             if expected_error in line and "failed" in line]
```

Reply:
cleaned up